exercises-toolbox/3-matplotlib/5/loesung.py

```diff
@@ -14,13 +14,7 @@
 
 plt.xticks(
     np.arange(0, 2.1 * np.pi, np.pi / 2),
-    [
-        r"$0$",
-        r"$\frac{1}{2}\pi$",
-        r"$\pi$",
-        r"$\frac{3}{2}\pi$",
-        r"$2\pi$",
-    ],
+    [r"$0$", r"$\frac{1}{2}\pi$", r"$\pi$", r"$\frac{3}{2}\pi$", r"$2\pi$"],
```

Comment:
Why do you prefer this long line? If I remember correctly I used a formatter for this (black).

Reply:
I did the same, don't know why it changed it then.

Reply:
Black updates sometimes change formatting. It seems black decided that since it fits on one line, it should...

Reply:
But in regard to the readibilty for beginners I would keep the old version.

Reply:
I thought that black does not reformat lists if they have trailing commas. Did you maybe remove the trailing comma before running black?

Reply:
I only changed the layout line.